### PR TITLE
docs: Update Alan's PPP

### LINKED ISSUE #153: Update PPP: Alan
Update PPP Draft

https://nus-cs2103-ay2324s1.github.io/website/schedule/week12/project.html#4-draft-the-ppp

|Content|Recommended (PDF pages)|Hard Limit (PDF pages)|
|---|---|---|
|Overview + Summary of contributions|0.5-1|2|
[Optional] Contributions to the User Guide|1| |
[Optional] Contributions to the Developer Guide|3| |

### PR DESCRIPTION
**Pull request by freshcabbage123**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [X] Documentation Update

## Description

-   Update PPP

## Related Tickets & Documents

-   Closes #153 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [ ] Yes
-   [X] No, and this is why: Docs
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

**Comment by freshcabbage123** [comment]
Open PR to appease tracker

**Review by m1oojv — APPROVED** [review]
LGTM

File: docs/team/freshcabbage123.md [hunk 1]
```diff
@@ -1,43 +1,64 @@
----
+[---
```
Comment (HugeNoob) [inline docs/team/freshcabbage123.md:1]: Why is there a new bracket?

Comment (jiakai-17) [reply docs/team/freshcabbage123.md:1]: I think the bracket is unintentional.

Comment (jiakai-17) [reply docs/team/freshcabbage123.md:1]:
```suggestion
---
```

**Review by HugeNoob — APPROVED** [review]
Merging first for the tracker. If the bracket is a mistake, you can fix it later.

**Pull request merged**